graylog2-server/src/test/java/org/graylog/plugins/views/search/rest/ViewsResourceTest.java

```diff
@@ -106,26 +107,30 @@ public class ViewsResourceTest {
     @Mock
     private SearchFilterVisibilityChecker searchFilterVisibilityChecker;
 
+    @Mock
+    private ReferencedSearchFiltersRetriever referencedSearchFiltersRetriever;
+
     private ViewsResource viewsResource;
 
     @BeforeEach
     public void setUp() {
-        this.viewsResource = new ViewsTestResource(viewService, clusterEventBus, userService, searchDomain);
+        this.viewsResource = new ViewsTestResource(viewService, clusterEventBus, userService, searchDomain, referencedSearchFiltersRetriever);
```

Comment:
If we make the retriever just a util class, we don't need to inject it anywhere :-)

Reply:
I'd love to keep it as normal class, not util class with static method. I hope it will be acceptable by you...